src/clib/lib/enkf/gen_kw.cpp

```diff
@@ -182,18 +182,7 @@ bool gen_kw_initialize(gen_kw_type *gen_kw, int iens, const char *init_file,
 
     if (init_file)
         ret = gen_kw_fload(gen_kw, init_file);
```

Comment:
This whole function should be possible to remove

Reply:
Removed. Also removed the fload function